cloudferrylib/os/actions/host_aggregation_transporter.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2015 Mirantis Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and#
+# limitations under the License.
+
+
+from cloudferrylib.base.action import transporter
+from cloudferrylib.utils import utils as utl
+
+LOG = utl.get_log(__name__)
+
+class HostAggregationTransporter(transporter.Transporter):
```

Comment:
Please add docstring describing
 - what this task does
 - what are the required config options
 - what tasks depend on this task
 - what tasks require this task to be run